app/renderer/components/urlBarIcon.js

```diff
@@ -11,6 +11,19 @@ const dndData = require('../../../js/dndData')
 const {isSourceAboutUrl} = require('../../../js/lib/appUrlUtil')
 const searchIconSize = 16
 
+const getIconCssClass = (ctx) => {
+  if (ctx.isSearch) {
```

Comment:
still because it's not the issue which this PR tries to resolve, you might skip it. I'll take care of it then